virtcontainers/pkg/annotations/annotations.go

```diff
@@ -237,6 +237,9 @@ const (
 
 	// AgentTraceMode is a sandbox annotation to specify the trace type for the agent.
 	AgentTraceType = kataAnnotAgentPrefix + "trace_type"
+
+	// AgentContainerPipeSize is an annotation to specify the size of the pipes created for containers
+	AgentContainerPipeSize = kataAnnotAgentPrefix + "container_pipe_size"
```

Comment:
Since `container_pipe_size` is used outside this package, can you create a public `const` for this?

Reply:
Done!